runtime/onert/api/nnfw/src/nnfw_api_internal.cc

```diff
@@ -1833,3 +1839,109 @@ NNFW_STATUS nnfw_session::quantize()
 
   return NNFW_STATUS_NO_ERROR;
 }
+
+NNFW_STATUS nnfw_session::set_compiled_model_path(const char *path)
+{
+  try
+  {
+    if (!isStateModelLoaded())
+    {
+      std::cerr << "invalid state" << std::endl;
+      return NNFW_STATUS_INVALID_STATE;
+    }
+
+    assert(_codegen_manager != nullptr);
+    _codegen_manager->exportModelPath(std::string(path));
```

Comment:
This is just setter for `_export_model_path`, right? This is really confusing... In the future, this should be fixed, I think.

Reply:
(Above all, I don't say that this should be done right now. This is one of points to revise in the future.)

I talked to the name `exportModelPath()`.
```cpp
_codegen_manager->exportModelPath(std::string(path));
```

export is a verb word so the method can be translated to 'the manager exports with the model path and returns a something to be export'.

That's why I'm saying to fix it. 😄